doc/man7/ossl-ex-api.pod

```diff
@@ -0,0 +1,54 @@
+=pod
+
+=head1 NAME
+
+ASN1_STRING_data,
+ossl-norwegian-blue, ossl-pining-for-the-fjords, ossl-ex-api -
+API that has been removed from OpenSSL
+
+=head1 SYNOPSIS
+
+This page lists API that has been removed from OpenSSL, and when it was removed.
+
+=head1 DESCRIPTION
+
+This page documents a list of API that was removed from
+OpenSSL. Functions are not removed until they have been deprecated for
+some time (Normally at least one major OpenSSL release, if not
```

Comment:
Most of this paragraph could be removed and replaced by a link to removal.pod if removal.pod became a policy